### ISSUE TITLE
New Maintainer

### ISSUE THREAD

**Issue by joshuah** [joshuah/tarball-extract#10]
Mattie (@matsieftw)  will be taking over the project from now on.

**Comment by matsieftw** [comment]
Can you do an official transfer to me in the repo settings? @joshuah

**Comment by joshuah** [comment]
`matsieftw/tarball-extract already exists`, looks like you will need to delete or rename your copy for the transfer to work.

**Comment by matsieftw** [comment]
@joshuah You are correct. That's on me for not deleting my fork. I just did. You can transfer the repo to me now and then I will run an npm publish with the new location.

**Comment by joshuah** [comment]
Completed.

**Comment by joshuah** [comment]
Looks good but make sure you update the url field in package.json to point to your git repo URL.

**Comment by matsieftw** [comment]
hmm. Is `"url": "git://github.com/matsieftw/tarball-extract.git"` not correct?

Should it be the https url?

**Comment by joshuah** [comment]
NPM is leaving the old path for some reason. The package is installing properly. Github is redirecting the URL to your repo so it should be ok. You might want to do an install of the package on a clean directory and look into the npm warnings.
